Doc(eos_designs): Add lab topologies for all examples (#6483)
aristanetworks/ansible-avd: ansible_collections/arista/avd/examples/common/start-avd-playground.md

Reviewer: This file is not referenced anywhere that I can see. What is the purpose?

Author: The file is referenced in every example readme.

<!--
  ~ Copyright (c) 2024-2026 Arista Networks, Inc.
  ~ Use of this source code is governed by the Apache License 2.0
  ~ that can be found in the LICENSE file.
  -->

> AVD playground is an interactive Cloud-based lab environment sponsored by Arista.

Reviewer 2: If you intend this to be preview, you should say so either in this file or in every heading where you include it.

Author: as discussed, the preview status will be advertised in the lab readme

It's enough to authenticate on [labs.arista.com](https://labs.arista.com) and click the button below to start the lab.

[Start AVD Playground :octicons-play-16:](https://labs.arista.com/launch?lab_type=avd-playground&origin=tech-lib){ .md-button .md-button--primary target=_blank}

Please check [How to use AVD examples](../../../../../ansible_collections/arista/avd/examples/README.md) for details.

!!! note

>     AVD playgrounds are only available to registered Arista customers with support contract due to cost and security considerations. Please use AVD playground for good cause only and mind the EULA terms you signed during the registration. Do the right thing - be Arista customer! However if you don't have required level of access - talk to your Arista account team or [download the lab files](https://aristanetworks.github.io/aclabs/lab_archives/avd-playground.tar.gz) and run the playground locally on your own host. Unfortunately we will not be able to support this type of deployment officially due to number of issues that can exist on a random host machine, including missing Linux kernel configuration, misconfigured container engines, privilege issues, etc. The download option is for skilled users only that can manage the environment without extra support.

Reviewer 2: ```suggestion
    AVD playgrounds are only available to registered Arista customers with a support contract due to cost and security considerations.
    Please use AVD playground for good causes only and mind the EULA terms you signed during the registration.
    Do the right thing - be an Arista customer!
    However if you don't have the required level of access, talk to your Arista account team or
    [download the lab files](https://aristanetworks.github.io/aclabs/lab_archives/avd-playground.tar.gz)
    and run the playground locally on your own host.
    The download option is for skilled users only that can manage the environment without extra support.
```

<!--

>   ~ Copyright (c) 2026 Arista Networks, Inc.

Reviewer 3: I'm reviewing the language here and will push suggestions

Reviewer 2: If we have urgent needs to get the lab files merged, we could just move all the doc changes to a different PR, since it is the main concern right now.

Reviewer 3: reviewed with @JulioPDX and improved text.

>   ~ Copyright (c) 2024-2026 Arista Networks, Inc.

Reviewer 4: I guess this is a copy paste

```suggestion
  ~ Copyright (c) 2026 Arista Networks, Inc.
```

  ~ Use of this source code is governed by the Apache License 2.0
  ~ that can be found in the LICENSE file.
  -->

AVD playground is an interactive Cloud-based lab environment sponsored by Arista.

> It's enough to authenticate on [labs.arista.com](https://labs.arista.com) and click the button below to start the lab.

Reviewer 4: ```suggestion
To get started, simply sign in at [labs.arista.com](https://labs.arista.com/) and click the button below to spin up the lab.
```

Reviewer 3: I started reviewing and optimizing the language here. I will push a commit shortly!

  ~ Use of this source code is governed by the Apache License 2.0
  ~ that can be found in the LICENSE file.
  -->

AVD playground is an interactive Cloud-based lab environment sponsored by Arista.
It's enough to authenticate on [labs.arista.com](https://labs.arista.com) and click the button below to start the lab.

[Start AVD Playground :octicons-play-16:](https://labs.arista.com/launch?lab_type=avd-playground&origin=tech-lib){ .md-button .md-button--primary target=_blank}

Please check [How to use AVD examples](../../../../../ansible_collections/arista/avd/examples/README.md) for details.

!!! note

    AVD playgrounds are only available to registered Arista customers with a support contract due to cost and security considerations.
    Please use AVD playground for good causes only and mind the EULA terms you signed during the registration.
    Do the right thing - be an Arista customer!
    However if you don't have the required level of access, talk to your Arista account team or
    [download the lab files](https://aristanetworks.github.io/aclabs/lab_archives/avd-playground.tar.gz)
    and run the playground locally on your own host.

>     The download option is for skilled users only that can manage the environment without extra support.

Reviewer 2: ```suggestion
    However if you don't have the required level of access, talk to your Arista account team.
```
